generic: 4.19 and 5.4: Add 4B_OPCODES flag to w25q256

robimarko opened the pull request:
This patch backports the upstream patch that adds the 4B_OPCODES flag to w25q256 under 4.19 and 5.4 kernels.
This is needed for ipq40xx and ramips.

Signed-off-by: Robert Marko <robimarko@gmail.com>

robimarko commented:
@ynezz Any chance of merging this quickly.
I completely forgot to include it for kernel 5.4 as well.

981213 commented:
What about throwing this backport into target/linux/generic/backport-5.4? This is useful for other targets as well. There's a 4B read op patch in ramips for w25q256 and we should be able to get rid of that one in 5.4 with this backport.

dengqf6 commented:
This is not a platform-specific patch. Please move it to `generic/backport-5.4`

robimarko commented:
Ok, will move both the 4.19 version and 5.4 to generic ones.
I did not know that other platforms used the same patch.

robimarko committed:
generic: 4.19: Add 4B_OPCODES flag to w25q256

This patch backports the upstream patch that adds the 4B_OPCODES flag to w25q256 under 4.19 kernel.
This is needed for ipq40xx and ramips.

Signed-off-by: Robert Marko <robimarko@gmail.com>

robimarko committed:
generic: 5.4: Add 4B_OPCODES flag to w25q256

This patch backports the upstream patch that adds the 4B_OPCODES flag to w25q256 under 5.4 kernel.
This is needed for ipq40xx and ramips.

Signed-off-by: Robert Marko <robimarko@gmail.com>

robimarko commented:
Ok, I backported this to 4.19 and 5.4 in generic target

ynezz commented:
Thanks! Pulled into my staging tree at https://git.openwrt.org/openwrt/staging/ynezz.git
